### PR TITLE
GEOMESA-1625 Storm QS Updates

### PR DESCRIPTION
**Pull request by aheyne**
* Updated Docs for Storm Quickstart
* Fixed supported Accumulo versions
* Added Accumulo 1.8 profile to geomesa-accumulo-datastore.pom to shade curator to prevent conflicts with Accumulo 1.8 and Storm

Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-accumulo/geomesa-accumulo-datastore/pom.xml [hunk 1]
```diff
@@ -146,6 +146,60 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>accumulo-1.8</id>
+            <properties>
+                <accumulo.version>1.8.0</accumulo.version>
+                <thrift.version>0.9.3</thrift.version>
+            </properties>
```
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-datastore/pom.xml]: are these props not picked up by the profile in the main pom? having the versions in two places isn't ideal.

**Comment by elahrvivaz** [comment]
Would it be more appropriate to do the shading in whatever the storm client jar is? (e.g. in the tutorial, presumably) Seems like we wouldn't necessarily want to shade it here.

**Comment by elahrvivaz** [comment]
maybe we can just add a note to the documentation about the potential conflict

**Comment by aheyne** [comment]
Managing the curator version in the geomesa-tutorials solves this problem for the storm quickstart.

**Comment by aheyne** [comment]
Could someone flag this 1.3.0 please.

**Pull request merged**